components/ILIAS/Authentication/classes/class.ilAuthLoginPageEditorGUI.php

```diff
@@ -119,7 +135,7 @@ public function executeCommand(): void
                     '_top'
                 );
 
-                if ($this->redirect_source !== "ilinternallinkgui") {
+                if ($this->redirect_source !== 'ilinternallinkgui') {
```

Comment:
We should use `strtolower(ilInternalLinkGUI::class)` here.

```suggestion
                if (strtolower($this->redirect_source) !== strtolower(ilInternalLinkGUI::class)) {
```